### DISCUSSION TITLE
유니티에서 수정 이후 qr을 생성해도 이전 빌드가 유지되는 경우

### DISCUSSION THREAD

**Discussion by kimhaech** [naverz/zepeto-studio-kor#396] [5. Q&A]
제목 그대로 월드를 수정한 후 유니티 내에서 플레이를 하면 내용이 적용되지만 play on zepeto를 눌러 qr로 생성하는 경우 적용이 되질 않아서 테스트가 어렵네요. publish하여 테스트 코드를 받아서 하는 경우는 적용이 된 것 같은데 테스트 코드를 받아서 접속하는 방법은 안드로이드로 어려워서 상당히 번거롭게 되었습니다. 혹시 수정사항이 생기고 빌드 시 다른 작업을 진행 해야 함께 적용이 되는 것인지 질문 드립니다.

**Comment by runasy-koonta** [comment]
Platform이 Mobile로 설정이 되어 있는지 확인해보시겠어요?

**Reply by kimhaech** [reply]
현재 기본값으로 pc, 맥, 리눅스로 되어있습니다. 오늘 새로 빌드하니 적용이 되었긴 했는데 계속 안되는 프로젝트도 있네요!
기본 플랫폼이 모바일이 되어있어야하나요?

**Reply by runasy-koonta** [reply]
QR코드로 테스트 할 때, 선택된 플랫폼으로만 빌드되더라구요.
제 경우 Mobile로 되어 있어야 안드로이드, iOS가 빌드되어 QR 테스트시 정상적으로 동작했습니다.
